packaging/iofog-agent/usr/share/iofog-agent/upgrade.sh

```diff
@@ -51,6 +53,11 @@ get_distribution() {
 	esac
 
 	# Restore config and start agent
+	cd /var/backups/iofog-agent
+	tar -xzf config_backup$iofogpackage.tar.gz
+	mv etc/iofog-agent/* /etc/iofog-agent/
```

Comment:
Shouldn't this be `mv /var/backups/iofog-agent/* /etc/iofog-agent/`?

Reply:
It's already in the backup directory `cd /var/backups/iofog-agent` and when we untar the files are created in the `/var/backups/iofog-agent/etc/iofog-agent/` directory. 
That's why `mv etc/iofog-agent/* /etc/iofog-agent/`.